configs/default/DIAT-MAMBAF405US-I2C-WIFI.config

```diff
@@ -0,0 +1,124 @@
+# Betaflight / STM32F405 (S405) 4.1.0 Oct 16 2019 / 11:57:16 (c37a7c91a) MSP API: 1.42
+# manufacturer_id: DIAT   board_name: MAMBAF405US   custom defaults: NO
+
+board_name MAMBAF405US
+manufacturer_id DIAT
+
+# I/O
+resource LED 1 C15
+resource LED 2 C14
+resource BEEPER 1 C13
+set beeper_inversion = ON
+set beeper_od = OFF
+set osd_displayport_device = MAX7456
+
+# ACC/GYRO
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW180
+set gyro_1_align_yaw = 1800
+
+# SPI
+resource SPI_SCK 1 A05
+resource SPI_MISO 1 A06
+resource SPI_MOSI 1 A07
+
+resource SPI_SCK 2 B13
+resource SPI_MISO 2 B14
+resource SPI_MOSI 2 B15
+
+resource SPI_SCK 3 C10
+resource SPI_MISO 3 C11
+resource SPI_MOSI 3 B05
+
+# FLASH
+resource FLASH_CS 1 A15
+set flash_spi_bus = 3
+
+# OSD
+resource OSD_CS 1 B12
+set max7456_spi_bus = 2
+
+# ADC
+resource ADC_BATT 1 C01
+resource ADC_RSSI 1 C02
+resource ADC_CURR 1 C03
+set adc_device = 3
+
+# SERIAL PORTS
+resource SERIAL_TX 1 B06
+resource SERIAL_RX 1 B07
+
+resource SERIAL_TX 2 A02
+resource SERIAL_RX 2 A03
+
+resource SERIAL_TX 3 B10
+resource SERIAL_RX 3 B11
+
+resource SERIAL_TX 4 A00
+resource SERIAL_RX 4 A01
+
+resource SERIAL_TX 5 C12
+resource SERIAL_RX 5 D02
+
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 6 C07
+
+#I2C
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+
+# TIMERS
+timer B09 AF3   # pin B09: TIM11 CH1 (AF3)
+timer A09 AF1   # pin A09: TIM1 CH2 (AF1)
+timer A08 AF1   # pin A08: TIM1 CH1 (AF1)
+timer C09 AF3   # pin C09: TIM8 CH4 (AF3)
+timer C08 AF3   # pin C08: TIM8 CH3 (AF3)
+timer B08 AF2   # pin B08: TIM4 CH3 (AF2)
+timer B03 AF1   # pin B03: TIM2 CH2 (AF1)
+resource MOTOR 1 A09
+resource MOTOR 2 A08
+resource MOTOR 3 C09
+resource MOTOR 4 C08
+resource LED_STRIP 1 B03
+
+
+# DMA
+dma ADC 3 0     # ADC 3: DMA2 Stream 0 Channel 2
+dma pin A09 0   # pin A09: DMA2 Stream 6 Channel 0
+dma pin A08 0   # pin A08: DMA2 Stream 6 Channel 0
+dma pin C09 0   # pin C09: DMA2 Stream 7 Channel 7
+dma pin C08 0   # pin C08: DMA2 Stream 2 Channel 0
+dma pin B08 0   # pin B08: DMA1 Stream 7 Channel 2
+dma pin B03 0   # pin B03: DMA1 Stream 6 Channel 3
+
+# FEATURE
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature TELEMETRY
+feature OSD
+
+# SERIAL
+serial 0 64 115200 57600 0 115200
+serial 3 1 115200 57600 0 115200
+
+
+# EXTRAS
+resource INVERTER 1 C00
+resource PINIO 1 B00
+
+# master
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set mag_hardware = NONE
+set baro_hardware = AUTO
+set serialrx_provider = SBUS
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set motor_pwm_protocol = DSHOT600
```

Comment:
Will this target only be sold as an AIO offering with ESCs capable of Dshot on-board? If not please remove.